modules/php/States/PlayCardTrait.php

```diff
@@ -56,6 +60,11 @@ public function arg_playCard()
     if ($playRule == -1) {
       return ["count" => "All but 1"];
     }
+
+    $addInflation = Utils::getActiveInflation() ? 1 : 0;
+    $partyBonus = Utils::isPartyInPlay() ? 1 + $addInflation : 0;
```

Comment:
And here as well, you need to check for `getActivePartyBonus`?

Reply:
yes, of course, absolutely right